core/method/shared/to_s.rb

```diff
@@ -24,6 +24,13 @@
     @string.should =~ /\#bar/
   end
 
+  ruby_version_is "2.7" do
+    it "returns a String containing method arguments" do
+      m = MethodSpecs::Methods.new.method :two_req
+      m.send(@method).should =~ /\#two_req\(a, b\)/
```

Comment:
You can use `m.send(@method).should.include?("two_req(a, b)")`.
That way, there is no need to escape any character.